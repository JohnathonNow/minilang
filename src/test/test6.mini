var X := 1
var Y := nil

if X then
	print('Correct\n')
else
	print('Wrong\n')
end

if Y then
	print('Wrong\n')
else
	print('Correct\n')
end

var S := "Hello world!"

if var Z := S:find("llo") then
	print('Z = {Z}\n')
else
	print('Wrong\n')
end

if var Z := S:find("elo") then
	print('Wrong\n')
elseif var W := S:find("wor") then
	print('W = {W}\n')
else
	print('Wrong\n')
end

