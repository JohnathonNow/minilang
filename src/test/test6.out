Correct
Correct
Z = 3
W = 7
